### ISSUE TITLE
Remove dependency on com.sun:tools

### ISSUE THREAD

**Issue by h908714124** [google/compile-testing#134]
Otherwise, it seems it cannot work on JDK 9

**Comment by ptahchiev** [comment]
I have the same issue - unable to build my project because of:
````
[INFO] ------------------------------------------------------------------------
[INFO] Building Platform Core 2.0.0.BUILD-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.729 s
[INFO] Finished at: 2017-11-03T15:32:32+02:00
[INFO] Final Memory: 34M/113M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project nemesis-platform-core: Could not resolve dependencies for project com.nemesis.platform:nemesis-platform-core:jar:2.0.0.BUILD-SNAPSHOT: Could not find artifact com.sun:tools:jar:9 at specified path /usr/lib/jvm/java-9-oracle/../lib/tools.jar -> [Help 1]

````

**Comment by netdpb** [comment]
We're working on this. It's really a dependency on tools.jar, which doesn't exist in JDK9.

**Comment by ptahchiev** [comment]
For me it just worked when I excluded the tools.jar:
````
        <dependency>
            <groupId>com.google.testing.compile</groupId>
            <artifactId>compile-testing</artifactId>
            <scope>test</scope>
            <exclusions>
               <exclusion>
                  <groupId>com.sun</groupId>
                  <artifactId>tools</artifactId>
               </exclusion>
            </exclusions>
        </dependency>
````

**Comment by netdpb** [comment]
This issue is fixed by c2f3634bcc10a484c1fea3b6297bd6006067977d, which will be in the next release.
